.github/labeler.yml

```diff
@@ -2,3 +2,4 @@
 # Used by https://github.com/department-of-veterans-affairs/va.gov-platform-architecture/blob/main/.github/workflows/pull_request_labeler.yml
 RFC:
 - rfc/**/*
+- '!rfc/README.md'
```

Comment:
this might need to be wrapped in brackets like example here:
https://github.com/actions/labeler#common-examples

Reply:
This part seems to indicate that the label would get applied to anything that matches either filter?
> From a boolean logic perspective, top-level match objects are OR-ed together and individual match rules within an object are AND-ed. Combined with ! negation, you can write complex matching rules.
We can merge and see what happens :)